src/blas/impl/KokkosBlas3_gemm_impl.hpp

```diff
@@ -99,11 +99,11 @@ struct impl_deep_copy_matrix_block<TeamHandle,ViewTypeScratch,ViewType,Layout,bl
     } else {
       Kokkos::parallel_for(Kokkos::TeamThreadRange(team,blockDim_j), [&] (const int j) {
 #ifndef KOKKOS_IMPL_BATCHED_GEMM_GCC_CXX14_WORKAROUND
-        const int idx_j = offset_j+j;
+        int idx_j = offset_j+j;
```

Comment:
FYI for other readers: See https://github.com/kokkos/kokkos-kernels/issues/543#issuecomment-566723360 .